README.md

```diff
@@ -27,7 +27,21 @@
 
 #### Steps to run
 
-- Update the connection string in appsettings.json in SimplCommerce.WebHost
+- Update the connection string either using:
+    - User Secrets Command Line Tool
+    Run the following command
+    ```
+    dotnet user-secrets set "ConnectionStrings:DefaultConnection" "Your ConnectionString"
+    ```
+    - User Secrets Tool for Visual Studio
+        1. Right click the SimplCommerce.WebHost 
+        2. Choose "Manage User Secrets"
+        3. Write Your secrets in `secrets.json` as the following:
+        ```
+        {
+            "ConnectionStrings:DefaultConnection": "Your ConnectionString"
+        }
+        ```
```

Comment:
I think we shouldn't make it more complicated to user. I prefer to make thing as simple as possible. Advanced user will know how to use Secrets if they want to

Reply:
This not complicated while I update the `README.md`, furthermore this is good for security, the user shouldn't expose any kind of secrets